src/server/routes/admin/nonces.ts

```diff
@@ -0,0 +1,169 @@
+import { Static, Type } from "@sinclair/typebox";
+import { FastifyInstance } from "fastify";
+import { StatusCodes } from "http-status-codes";
+import { eth_getTransactionCount, getRpcClient } from "thirdweb";
+import {
+  lastUsedNonceKey,
+  recycledNoncesKey,
+  sentNoncesKey,
+  splitLastUsedNonceKey,
+} from "../../../db/wallets/walletNonce";
+import { getChain } from "../../../utils/chain";
+import { normalizeAddress } from "../../../utils/primitiveTypes";
+import { redis } from "../../../utils/redis/redis";
+import { thirdwebClient } from "../../../utils/sdk";
+import { standardResponseSchema } from "../../schemas/sharedApiSchemas";
+import { walletWithAddressParamSchema } from "../../schemas/wallet";
+
+export const responseBodySchema = Type.Object({
+  result: Type.Array(
+    Type.Object({
+      walletAddress: Type.String({
+        description: "Backend Wallet Address",
+        examples: ["0xcedf3b4d8f7f1f7e0f7f0f7f0f7f0f7f0f7f0f7f"],
+      }),
+      onchainNonce: Type.String({
```

Comment:
IMO use numbers as nonces. They're already numbers throughout the code, they shouldn't be larger than INT_MAX, and they're more sort-able.

Reply:
will have to parse all of them because redis returns strings. I want to be consistent between redis nonce and onchain nonce so either all strings or all numbers. I am fine with doing all numbers, its just needing to parse the returned set.

Reply:
I think we should parse them then. I predict we'll want them sortable on the frontend. And visually (like when I curl the endpoint) I'd want to see them sorted too. Rather do that on the backend if it's going to be a common need.